lib/datadog/open_feature/binding/evaluator.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative 'resolution_details'
+
+module Datadog
+  module OpenFeature
+    module Binding
+      class Evaluator
+        def initialize(ufc_json)
+          # NOTE: In real binding we will parse and create Configuration
+          @ufc_json = ufc_json
+        end
+
+        def get_assignment(_flag_key, _evaluation_context, expected_type, _time)
+          ResolutionDetails.new(
+            value: generate(expected_type),
+            reason: 'TARGETING_MATCH',
+            variant: 'hardcoded-variant',
+            allocation_key: 'hardcoded-allocation-key',
+            flag_metadata: {
+              'doLog' => true,
+              'allocationKey' => 'hardcoded-allocation-key'
+            },
+            do_log: true,
+            extra_logging: {}
+          )
+        end
+
+        private
+
+        def generate(expected_type)
```

Comment:
this looks like a function for test purposes. If so, should it be moved from library code to spec support?

Reply:
It's a mock up